rfcs/0000-centralized-configuration-management.md

```diff
@@ -0,0 +1,384 @@
+
+## Status
+[Status]: #status
+
+* Status: Proposed
+* Implementation JIRA Ticket(s):
+    * [BFD-1639](https://jira.cms.gov/browse/BFD-1639)
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+* [RFC Proposal](#rfc-proposal)
+* [Status](#status)
+* [Table of Contents](#table-of-contents)
+* [Motivation](#motivation)
+* [Proposed Solution](#proposed-solution)
+    * [Proposed Solution: Detailed Design](#proposed-solution-detailed-design)
+    * [Proposed Solution: Unresolved Questions](#proposed-solution-unresolved-questions)
+    * [Proposed Solution: Drawbacks](#proposed-solution-drawbacks)
+    * [Proposed Solution: Notable Alternatives](#proposed-solution-notable-alternatives)
+* [Prior Art](#prior-art)
+* [Future Possibilities](#future-possibilities)
+* [Addendums](#addendums)
+
+## Motivation
+[Motivation]: #motivation
+
+Despite BFD's relatively modest infrastructure footprint, the existing configuration strategy is complex.
+How, where, and _when_ configuration is used informs us of the tools and locations of supporting code that may need to be updated to accommodate a change.
+Supported use-cases like renaming or adding new configuration may require one or more updates to the following locations:
+
+1. packer definitions
+2. sensitive ansible variables protected with `ansible-vault`
+3. nonsensitive ansible variables in plain-text
+4. ansible build playbooks
+5. ansible launch playbooks
+6. ansible role variables
+7. ansible role definitions
+8. sensitive terraform input variables stored as tfvars files in `keybase`
+9. nonsensitive terraform input variables passed between Jenkins job stages
+10. terraform templated cloud-init user data definitions
+11. terraform module definitions
+12. nonsensitive Jenkins job definitions stored in Jenkins
+13. nonsensitive Jenkins job definitions stored in various Jenkinsfiles
+14. sensitive Jenkins definitions stored with the Jenkins credential plugin
+
+The ability to supply configuration to each tool should be an enabling feature.
+Instead, the sprawling configuration has historically discouraged us from adopting worthy changes, contributed to operators' inadvertent addition of errors, and represents an unfriendly introduction to new contributors who wish to make what would otherwise be simple updates to BFD configuration.
+
+## Proposed Solution
+[Proposed Solution]: #proposed-solution
+
+AWS Systems Manager (SSM) Parameter Store is a hierarchical, key-value data store typically used for centrally managed sensitive and nonsensitive configuration data.
+In addition to the AWS Console, the Parameter Store is accessed through the Parameter Store API, and can be even more accessible through various tools, including:
+- the AWS CLI via `aws ssm` sub-command
+- terraform resources and data sources with the aws provider
+- ansible workflows through the `community.aws.aws_ssm_parameter_store` module
```

Comment:
Wouldn't it be more `https://docs.ansible.com/ansible/2.5/plugins/lookup/aws_ssm.html`?